airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/435bb9a5-7887-4809-aa58-28c27df0d7ad.json

```diff
@@ -2,7 +2,7 @@
   "sourceDefinitionId": "435bb9a5-7887-4809-aa58-28c27df0d7ad",
   "name": "MySQL",
   "dockerRepository": "airbyte/source-mysql",
-  "dockerImageTag": "0.3.9",
+  "dockerImageTag": "0.4.0",
```

Comment:
FWIW we can do `0.3.10` -- we don't have a real convention yet since we're in alpha still but I tend to think of it as: 

while in `0.X.X` we do minor for backwards breaking, patch for everything else
After `1.X.X` we do semver

Reply:
@sherifnada ah alright, was not aware of that. Do you want me to change this to `0.3.10`?

Reply:
no need